documentation/source/user_section/tutorials/lesion-analysis/lesion-segmentation/before-starting.rst

```diff
@@ -0,0 +1,20 @@
+Before starting this tutorial
+#############################
+
+#. Make sure that you have the following files in your working directory:
+
+   * ``single_subject/data/t2_lesion/t2_seg.nii.gz``: A 3D binary mask of the spinal cord.
+   * ``single_subject/data/t2_lesion/t2_lesion.nii.gz``: A 3D binary mask of the lesion.
```

Comment:
I don't understand why someone would need these files if they are actually trying to segment the SC or lesions. What is the point of these files ?

Reply:
For me this should be there for the second part of the tutorials "Compute lesion morphometric measures using sct_analyze_lesion"

Reply:
Good catch! I forgot to update this rst file. Updated in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4586/commits/a954cc0cd2232bb115a89c17f0758f08482dab6b